lib/modules/datasource/pypi/index.ts

```diff
@@ -182,10 +182,14 @@ export class PypiDatasource extends Datasource {
     // pep-0427 wheel packages
     //  {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl.
     const wheelText = text.toLowerCase();
-    const wheelPrefix = packageName.replace(regEx(/[^\w\d.]+/g), '_') + '-';
```

Comment:
Did this `/g` make sense if the regex starts with `^`?

Reply:
Yes I think so.  The `^` is inside `[]` so it is just negating the character classes `\w`, `\d`, and the character `.`, instead of denoting the start of string if it were before the `[`.  The `\g` is there to not return after the first match.  Otherwise you would end up with `test-package-name` changing into `test_package-name`?

Reply:
Thanks